include/SDL3/SDL_ioasync.h

```diff
@@ -0,0 +1,500 @@
+ * might take longer than the life of the calling function to complete!
+ *
+ * An SDL_IOAsyncGroup must be specified. The newly-created SDL_IOAsyncTask
+ * will be added to it. A group is used to query multiple async tasks at once.
+ *
+ * \param ioasync a pointer to an SDL_IOAsync structure.
+ * \param ptr a pointer to a buffer to write data from.
+ * \param offset the position to start writing to the data source.
+ * \param size the number of bytes to write to the data source.
+ * \param group a group to add the new SDL_IOAsync to.
+ * \param userdata an app-defined pointer that will be provided with the task results.
+ * \returns A new task handle if a task was started, NULL on complete failure;
+ *          call SDL_GetError() for more information.
+ *
+ * \threadsafety It is safe to call this function from any thread.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_ReadIOAsync
+ * \sa SDL_CreateIOAsyncGroup
+ * \sa SDL_GetIOAsyncTaskResult
+
+ */
+extern SDL_DECLSPEC SDL_IOAsyncTask * SDLCALL SDL_WriteIOAsync(SDL_IOAsync *ioasync, void *ptr, Uint64 offset, Uint64 size, SDL_IOAsyncGroup *group, void *userdata);
+
+/**
+ * Close and free any allocated resources for an async I/O object.
+ *
+ * Closing a file is _also_ an asynchronous task! If a write failure
+ * were to happen during the closing process, for example, the
+ * task results will report it as usual.
+ *
+ * This function guarantees that the close will happen after any other
+ * pending tasks to `ioasync`, so it's safe to open a file, start
+ * several operations, close the file immediately, then check for all
+ * results later. This function will not block until the tasks have
+ * completed.
+ *
+ * Once this function returns non-NULL, `ioasync` is no longer valid,
+ * regardless of any future outcomes. Any completed tasks might still
+ * contain this pointer in their SDL_IOAsyncOutcome data, in case the
+ * app was using this value to track information, but it should not
+ * be used again.
+ *
+ * If this function returns NULL, the close wasn't started at all, and
+ * it's safe to attempt to close again later.
+ *
+ * \param ioasync a pointer to an SDL_IOAsync structure to close.
+ * \param group a group to add the new SDL_IOAsync to.
+ * \param userdata an app-defined pointer that will be provided with the task results.
+ * \returns A new task handle if a task was started, NULL on complete failure;
+ *          call SDL_GetError() for more information.
+ *
+ * \threadsafety It is safe to call this function from any thread, but
+ *               two threads should not attempt to close the same object.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC SDL_IOAsyncTask * SDLCALL SDL_CloseIOAsync(SDL_IOAsync *ioasync, SDL_IOAsyncGroup *group, void *userdata);
+
+/**
+ * Create a task group for tracking multiple I/O operations.
+ *
+ * An async I/O operation can be assigned to a group when started. The
```

Comment:
can be assigned -> must be assigned

Reply:
Fixed in 5829b1ff4fb0b9859c0f46e3eb0dc7f4ce5f5b7a.